NEWS

```diff
@@ -1,4 +1,6 @@
 v0.8.18-head 2014-XX-YY  The Irssi team <staff@irssi.org>
+	+ Garbage Collection support has been removed. This will hardly have any
+	  affect for anyone given that it has been unsuported for several years.
```

Comment:
s/affect/effect/, s/unsuported/unsupported/

(pretty much the only thing that can be reviewed here, really)